app/modules.go

```diff
@@ -110,7 +111,6 @@ var MaccPerms = map[string][]string{
 	// MilkyWay permissions
 	rewardstypes.RewardsPoolName:  nil,
 	liquidvestingtypes.ModuleName: {authtypes.Minter, authtypes.Burner},
-	investorstypes.ModuleName:     nil,
```

Comment:
You are removing this since we don't anymre funds to this module account correct?

Reply:
Lets keep as it is to avoid any problem